lib/CFRMessageProvider.jsm

```diff
@@ -354,6 +354,99 @@ const CFR_MESSAGES = [
     frequency: {lifetime: 3},
     trigger: {id: "frequentVisits", params: PINNED_TABS_TARGET_SITES},
   },
+  {
```

Comment:
You could actually maybe add this to the `SnippetsTestProvider` (although perhaps renaming it to make it more generic would make sense)

Reply:
Although I guess we wouldn't want this to always show up during regular debugging...